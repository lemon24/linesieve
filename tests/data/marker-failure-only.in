--section '(\S+):$' --failure fail show '$nothing'
---
one:
1
two:
2
fail
