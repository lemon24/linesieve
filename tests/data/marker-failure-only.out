..
two
2
fail
